### ISSUE TITLE
Error when running without snapping

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/rapid-watershed-delineation#14]
Passing a `0` to the RWD function throws the following exception:

```
Error running Point_Watershed_Function
Traceback (most recent call last):
  File "/opt/rwd/src/api/main.py", line 60, in run_rwd
    output_path
  File "/opt/rwd/rwd/Rapid_Watershed_Delineation.py", line 132, in Point_Watershed_Function
    define_projection('mypoint', 'New_Outlet', infile_crs[0])
  File "/opt/rwd/rwd/RWSDelin_Utilities.py", line 149, in define_projection
    with fiona.open(input_shapefile1) as source:
  File "/usr/local/lib/python2.7/site-packages/fiona/__init__.py", line 162, in open
    raise IOError("no such file or directory: %r" % path)
IOError: no such file or directory: 'mypoint.shp'
```

**Comment by mmcfarland** [comment]
[This commit](https://github.com/nazmussazib/RapidWatersheDelineation/commit/9af76718afaa593a5b015951bb3b8ecb37d5e4d5) needs to get ported from upstream and a new release cut for the non-snapping to be fixed.

**Comment by mmcfarland** [comment]
Fixed in #15
